roles/eda/vars/main.yml

```diff
@@ -14,3 +14,9 @@ eda_event_stream_prefix_path: "{{ event_stream.prefix | default('/eda-event-stre
 event_stream_nginx_port: 8000
 event_stream_server_name: "{{ ansible_operator_meta.name }}-event-stream"
 event_stream_django_port: 8002
+
```

Comment:
@ttuffin I ended up moving this out of defaults over to vars, it seems like this is the more "correct" spot for these based on the other variables present in this file that get used in the `gunicorn --bind` statements. If the vars are here it works as I expect in terms of correctly setting the vars in the deployment/pod.